public/lib/analytics.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Analytics module which listens for events and tracks them as appropriate
+ * within Google analytics.
+ */
+
+import angular from 'angular';
+
+const initGA = gaId => {
+    // tracking script should be on the page already
+    if (gaId) {
+        window.ga =
+            window.ga ||
+            ((...args) => (window.ga.q = window.ga.q || []).push(args));
+
+        const { ga } = window;
+        ga("create", gaId, "auto");
+        ga("set", "transport", "beacon");
+        ga("send", "pageview");
+
+        return ga;
+    }
+    return (...args) => window.debugGA && console.log(...args);
+};
+
+
+const wfAnalyticsServiceMod = angular.module('wfAnalyticsServiceMod', []);
+
+wfAnalyticsServiceMod.service('wfAnalyticsService', [
+    '$rootScope',
+    function($rootScope) {
+
+        //setup ga
+        const gaId = 'UA-78290349-2';
```

Comment:
Is it safe to put this in an open repo?

Reply:
It was only supposed to be here temporarily but I forgot to move it. It's not unsafe as you can find it out from the network tab on any page anyway, and all anyone can do is send analytics to our account, but it's not ideal in terms of keeping config in one place. I'll move it over.